src/components/TextInput/TextInput.tsx

```diff
@@ -193,11 +176,7 @@ export const TextInput: ChakraComponent<
       const finalInvalidText = invalidText
         ? invalidText
         : "There is an error related to this field.";
-      const internalOnChange = (
-        e:
-          | React.ChangeEvent<HTMLInputElement>
-          | React.ChangeEvent<HTMLTextAreaElement>
-      ) => {
+      const internalOnChange = (e) => {
```

Comment:
I think you should still type this, and since this is now the native html function, it can be use the html type.

Reply:
Hmm, if I add the types back in, Typescript complains bc it doesn't know whether the `onChange` is the `onChange` for input or for text area.

```
Argument of type 'ChangeEvent<HTMLTextAreaElement> | ChangeEvent<HTMLInputElement>' is not assignable to parameter of type 'ChangeEvent<HTMLInputElement>'.
```

Do you know how to fix?

Reply:
No, unfortunately that causes the same error. Typescript isn't positive that the correct type of event will be passed for the onChange the function has access to.

Reply:
:( ok to leave as is.